sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -127,11 +127,11 @@ def begin_create_certificate(
 
     @distributed_trace
     def get_certificate(self, certificate_name, **kwargs):
-        # type: (str, Any) -> KeyVaultCertificate
+        # type: (str, **Any) -> KeyVaultCertificate
         """Gets a certificate with its management policy attached. Requires certificates/get permission.
 
-        Does not accept the version of the certificate as a parameter. If you wish to specify version,
-        use the :func:`get_certificate_version` function and specify the desired version.
+        Does not accept the version of the certificate as a parameter. To get a specific version of the
+        certificate, use the :func:`get_certificate_version` function and specify the desired version.
```

Comment:
Brevity nitpick: I'd write "To get a specific version of the certificate, call :func:get_certificate_version." (Backticks around the method name not included because I don't know how to escape them.)

Reply:
sounds better, thanks!